### PR TITLE
2441 - (follow up)make field name more descriptive

### PR DESCRIPTION
**Pull request by ywrsusan**
*Issue number of the reported bug or feature request: #<2441>*

### CODE REVIEW

File: templates/projects/project.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
                     <td><p>{{n_available_tasks}}</p></td>
                 </tr>
                 <tr>
-                    <td><p>Total number of task runs:</p></td>
+                    <td><p>Total number of expected task runs (include gold tasks):</p></td>
```
Comment (fiorda) [inline templates/projects/project.html]: I'm confused now, how can the total number of expected task runs include gold tasks?

Comment (ywrsusan) [reply templates/projects/project.html]: I mean, it includes gold task runs that already happened

Comment (fiorda) [reply templates/projects/project.html]: ok

Comment (ywrsusan) [reply templates/projects/project.html]: maybe change to "Total number of expected task runs (include answered gold task runs)" ?

**Pull request closed without merging**